src/components/PostPageDetail/index.js

```diff
@@ -585,7 +585,7 @@ const PostPageDetailIdComponent = (props) => {
             isBackButton={true}
             source={SOURCE_PDP}
             height={getHeightHeader()}
-            isFollow={item?.is_following_target || parentData?.is_following_target}
+            isFollow={item?.is_following_target}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made here simplifies the logic for setting the `isFollow` prop by removing the dependency on `parentData?.is_following_target`. This is a good change if `item?.is_following_target` is guaranteed to always have the correct follow status. However, if there are cases where `item?.is_following_target` might be undefined or not yet updated and `parentData?.is_following_target` is used as a fallback, this change could introduce bugs. Please ensure that `item?.is_following_target` will always provide the correct follow status.
```diff
- isFollow={item?.is_following_target || parentData?.is_following_target}
+ isFollow={item?.is_following_target}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->